controllers/datadogagent/feature/cspm/feature.go

```diff
@@ -64,11 +66,11 @@ func (f *cspmFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.Req
 		}
 
 		if dda.Spec.Features.CSPM.CustomBenchmarks != nil {
-			f.configMapName = dda.Spec.Features.CSPM.CustomBenchmarks.Name
-			f.configMapConfig = dda.Spec.Features.CSPM.CustomBenchmarks
+			f.customConfig = v2alpha1.ConvertCustomConfig(dda.Spec.Features.CSPM.CustomBenchmarks)
 		}
+		f.configMapName = apicommonv1.GetConfName(dda, f.customConfig, apicommon.DefaultCSPMConf)
 
-		// TODO add settings to configure f.createSCC and f.createPSP
+		// CELENE TODO add settings to configure f.createSCC and f.createPSP
```

Comment:
IMO it should be a global config for that

Reply:
👍 i'll let @khewonc make this call